sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -113,6 +113,7 @@ add_sycl_plugin(level_zero
     "../unified_runtime/ur/adapters/level_zero/program.hpp"
     "../unified_runtime/ur/adapters/level_zero/queue.hpp"
     "../unified_runtime/ur/adapters/level_zero/sampler.hpp"
+    "../unified_runtime/ur/adapters/level_zero/physical_mem.hpp"
```

Comment:
minor: could you put the new entries in alphabetical order?

Reply:
They should all be in alphabetical order now, unless I've missed some.